roles/config-rh-sso/README.md

```diff
@@ -0,0 +1 @@
+WIP
```

Comment:
Would like to see at least some minimal README information here saying at least what this is and what it does.